src/scripts/main.js

```diff
@@ -1,3 +1,260 @@
+/* eslint-disable max-len */
 'use strict';
 
-// write code here
+const QUERY = {
+  'Name': 0,
+  'Position': 1,
+  'Office': 2,
+  'Age': 3,
+  'Salary': 4,
+};
+
+const queryName = 0;
+const queryPosition = 1;
+const queryOffice = 2;
+const queryAge = 3;
+const querySalary = 4;
+
+const notificationSuccess = 'success';
+const notificationError = 'error';
+
+const body = document.querySelector('body');
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const employees = tbody.children;
+
+let desc = false;
+let query = '';
+
+function sortEmployees(sortedQuery) {
+  switch (sortedQuery) {
+    case queryName:
+    case queryPosition:
+    case queryOffice:
+      return desc
```

Comment:
Let's improve your solution
- don't need to write logic for asc and desc in every case. 
- you can create variable, set sorters list to this variable 
- if sort type is desc you need to rerun a reversed sorted list (with method revers)
- if sort type is asc than just return a sorted list